tccli/argparser.py

```diff
@@ -51,6 +51,8 @@ def _check_value(self, action, value):
                 for word in possible:
                     extra.append('  * %s' % word)
                 msg.extend(extra)
+            msg.append("If invalid choice doesn't include the action or product you input, " \
```

Comment:
Invalid choice: The specified command or option is not found, try lastest version by running `pip install --upgrade tccli`.